### PR TITLE
Docs: update protobuf guideline pages

### PR DESCRIPTION
**Pull request by Mo3et**
Update `api/protobuf`  pages && Fix some outdated contents.

### CODE REVIEW

**Comment by Mo3et** [comment]
# Review Link 

 [Update Docs CN](https://kratos.caitou.org/docs/guide/api-protobuf) && [Update Docs EN](https://kratos.caitou.org/en/docs/guide/api-protobuf)

[Origin Docs CN](https://go-kratos.dev/docs/guide/api-protobuf/) && [Origin Docs EN](https://go-kratos.dev/en/docs/guide/api-protobuf/)

File: docs/guide/01-api-protobuf.md [hunk 1]
```diff
@@ -13,78 +13,94 @@ keywords:
...
+项目中定义 Proto，以 api 为包名根目录：
 
-项目中定义Proto，以api为包名根目录：
 ```bash
 kratos-demo：
 |____api // 服务API定义
 | |____kratos
 | | |____demo
-| | | |____v1
+| | | |____v2
```
Comment (Windfarer) [inline docs/guide/01-api-protobuf.md:31]: why v2

Comment (Mo3et) [reply docs/guide/01-api-protobuf.md]: Sorry. I have some error understand. I'm reverted it.

File: docs/guide/01-api-protobuf.md [hunk 2]
```diff
@@ -111,46 +132,61 @@ Java:
...
 
-### Examples
-API Service接口定义(demo.proto)
 ```protobuf
 syntax = "proto3";
 
 package kratos.demo.v1;
 
 // 多语言特定包名，用于源代码引用
-option go_package = "github.com/go-kratos/kratos/demo/v1;v1";
```
Comment (Windfarer) [inline docs/guide/01-api-protobuf.md:153]: the name after ; is for relative code generation, should not be removed

Comment (Mo3et) [reply docs/guide/01-api-protobuf.md:153]: OK. I understands. Shoud I need add this comment to the code?

Comment (Mo3et) [reply docs/guide/01-api-protobuf.md:153]: I'm update it and add the comment. If you think the comment need to be removed. pls comment it. Thx!

**Review by Windfarer — APPROVED** [review]

**Pull request merged**